jenkins/cori_atmnbfb.sh

```diff
@@ -5,5 +5,11 @@ export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
 export CIME_MACHINE=cori-knl
 source $SCRIPTROOT/util/setup_common.sh
 
-module load python/2.7-anaconda-5.2
+# Unload python module from:
+# $SCRIPTROOT/util/setup_common.sh --> $SCRIPTROOT/util/${CIME_MACHINE}_setup.sh
+module unload python/2.7-anaconda-5.2
```

Comment:
@jgfouca I'm not 100% sure this is needed...

On Oct. 26, 2018 you added the `module load python/2.7-anaconda-5.2` command to this script in ca7409bb9d7ee13c1d1addf0d7733e91636fa9d1 , but I think that module should have been already loading via 0c020345847bb5c5224d8b06d45b9316409eea2f which added that module to the `cori-knl_setup.sh` script on Oct. 17, 20188. Is there a reason it was needed twice? (The network graph doesn't show the `cori_atmnbfb` jenkins job coming in from a branch that didn't have 0c020345847bb5c5224d8b06d45b9316409eea2f in it)

Reply:
You're right. It's not needed.

Reply:
@jgfouca sounds good -- I'll leave the unload then to remove the python coming from `cori-knl_setup.sh`

Thanks!